### DISCUSSION TITLE
plans for Draggable?

### DISCUSSION THREAD

**Discussion by yureckey** [primefaces/community#431] [PrimeVue]
I remember I read in Discord that there was an intent to release something with Draggable functionality for PrimeVue before the end of the year. Is this still on? 
(My wishlist is to be able drag&drop tree nodes, with events and filtering, so I can control what and where user can drag & drop)

**Comment by tugcekucukoglu** [comment]
It's on our roadmap. 

Thank you,

**Comment by chris-deep** [comment]
> I remember I read in Discord that there was an intent to release something with Draggable functionality for PrimeVue before the end of the year. Is this still on? (My wishlist is to be able drag&drop tree nodes, with events and filtering, so I can control what and where user can drag & drop)

I'll watch this thread but within the next month I will need to add drag drop to the PrimeVue tree.  I'll post here with what I learn / try...

**Comment by Tomekslib** [comment]
Please , relase it ! 
Thank you

**Comment by wijzijnweb** [comment]
Any news on this feature? @tugcekucukoglu
